### ISSUE TITLE
Add inputs and outputs to Logo

### ISSUE THREAD

**Issue by vegantriathlete** [LearningDevOps/intro-to-devops#5]
I love the initial concept and its minimalist design! I think the meshed gears do a great job of conveying automation.

I'm not totally sure what the circumscribing elements represents. But, it could be the "continuous" in the Continuous Integration / Continuous Development / Continuous Deployment.

I think what could help emphasize this is to have some elements around (they could be placed on) the circle to show what's happening in the cycle. Let's stick with the minimalist imagery and find some simple icons to represent:

A developer committing code
Tests running
A new build being created
An end user (utilizing the new build) and providing feedback

**Comment by vegantriathlete** [comment]
Example programmer icons:
https://openclipart.org/detail/211418/thenanobel-programming
https://openclipart.org/detail/211447/computer-user-warning-sign
https://openclipart.org/detail/188610/coder

**Comment by vegantriathlete** [comment]
Example robot icons to represent automated testing:
https://openclipart.org/detail/222620/cipy-rules
https://openclipart.org/detail/222618/cipy-geek
https://openclipart.org/detail/222617/cipy-thumb-up

**Comment by vegantriathlete** [comment]
Example robot icons to represent automated build:
https://openclipart.org/detail/20507/robot-carrying-things-1
https://openclipart.org/detail/151723/robot
https://openclipart.org/detail/223091/cipy-mail

**Comment by vegantriathlete** [comment]
Example package icons to represent automated build:
https://openclipart.org/detail/26101/cardboard-box-package
https://openclipart.org/detail/107719/package-applications

**Comment by vegantriathlete** [comment]
Example user icon: https://openclipart.org/detail/167201/user

Maybe have something like the above with a thought bubble.

This first thought bubble gives the idea I have of a rectangular type of bubble - https://openclipart.org/detail/167796/speech-1
This second one also gives a good idea of the shape of bubble I was thinking and I like the connector better - https://openclipart.org/detail/166610/messaggi
Even though there are two thought bubbles instead of one, they convey the idea pretty well. I prefer the grey one in the back since it's got a more complete 3D effect - https://openclipart.org/detail/129049/chat

**Comment by akshatgoel** [comment]
I think we should not add too many elements. I think we should start with this icon. Maybe I will enhance the elements already present but I don't think more elements will help the icon.

**Comment by vegantriathlete** [comment]
I understand your desire to keep things simple.

How about if you enhance the circle to be made up of arrows, similar to: http://superawesomevectors.com/wp-content/uploads/2013/05/arrow-circle-icon.jpg

**Comment by akshatgoel** [comment]
Already working on similar concept :)

**Comment by vegantriathlete** [comment]
I think it will turn out great, then! Looking forward to see the next version.

**Comment by akshatgoel** [comment]
V2 is now live!

**Comment by vegantriathlete** [comment]
That's an improvement.

I don't care whether we have two or three gears.
I'd like the "connectors" on the arrow heads to be thicker.
I'd prefer four arrows to three, but am willing to go with either one. It may make sense to have the number of arrows equal the number of gears, though. So, I guess that means I'd also want four gears and maybe you feel that's going to be too many elements.

If you like, I think it's worthwhile to post the image(s) to the Studio to get feedback. We could post just one and ask for their thoughts on it, or we could post several versions and ask which one they prefer. Are you interested / ready to get feedback from the Studio?

**Comment by akshatgoel** [comment]
I think number 3 provides more balance. Hence, 3 arrows and gears.

I was also thinking of posting the image to the studio for reviews. Do you think I should give 4 arrows (I think 4 gears will be overkill) a shot and then, post the two images?

**Comment by vegantriathlete** [comment]
Sure, why not see how they like the three arrows / three gears icon compared to the four arrows / three gears icon. I'd also like to get their feedback on thinner versus thicker connectors.

**Comment by akshatgoel** [comment]
How do you mean thicker connectors? The arrows or the circle?

**Comment by vegantriathlete** [comment]
I mean the circle. The arcs in between the arrow heads.

**Comment by akshatgoel** [comment]
I agree. That needs to be a bit thicker. Will do that once the concept is validated.

**Comment by vegantriathlete** [comment]
Excellent!

**Comment by akshatgoel** [comment]
Good feedback from Studio : Add transparent symbols over background to give more visual appeal to the image.

**Comment by vegantriathlete** [comment]
So are you thinking about incorporating some of the elements I had suggested previously?

**Comment by akshatgoel** [comment]
Yes. but in background as transparent symbols. I will have to find other icons though. Udemy doesn't like cliparts.

**Comment by vegantriathlete** [comment]
Sure, I understood that you meant transparent symbols in the background. Also, the examples were meant to be just that, examples. I didn't mean that you would copy and paste them.

As far as whether Udemy allows clipart, it's a pretty confusing guideline. I don't think we'd have a problem using clipart for the icons. Whether you create the icons yourself or copy them from a site like the one I referenced, I don't see how one would be classified as okay and the other as disallowed clipart. In either case, we'd be using some graphic icons. For example, why would the gears not be classified as clipart?

**Comment by vegantriathlete** [comment]
For example, look at the course image I created for https://www.udemy.com/command-line-survival-transfer-and-manage-files-with-ease/. That image has been explicitly approved.

I do think that many of the examples I included were too complicated and we want some simpler iconography. I chose that site because the images are free to use (although I didn't read the licensing terms).

Go ahead and keep working on the image however is most easy for you and don't get too hung up on the clipart guideline. We can send the image to instructorsupport (or instructorteam?) to have them review it for us while we continue to work on the rest of the course.
